ragnarok_formats/src/model.rs

```diff
@@ -57,21 +57,74 @@ where
 
 #[derive(Debug, ByteConvertable)]
 #[cfg_attr(feature = "interface", derive(korangar_interface::elements::PrototypeElement))]
-pub struct PositionKeyframeData {
+pub struct ScaleKeyframeData {
     pub frame: u32,
-    pub position: Point3<f32>,
+    pub scale: Vector3<f32>,
+    reserved: f32,
 }
-
 #[derive(Clone, Debug, ByteConvertable)]
 #[cfg_attr(feature = "interface", derive(korangar_interface::elements::PrototypeElement))]
 pub struct RotationKeyframeData {
     pub frame: u32,
     pub quaternions: Quaternion<f32>,
 }
 
+#[derive(Debug, ByteConvertable)]
+#[cfg_attr(feature = "interface", derive(korangar_interface::elements::PrototypeElement))]
+pub struct TranslationKeyframeData {
+    pub frame: u32,
+    pub translate: Vector3<f32>,
```

Comment:
Given that the struct is called `TranslationKeyframeData`, I think this should be `translation`